### ISSUE TITLE
33

### ISSUE THREAD

**Issue by fatmedamla1-boop** [fatmedamla1-boop/ai-programirane-#3]
import pandas as pd
from sklearn.feature_extraction.text import CountVectorizer
from sklearn.naive_bayes import MultinomialNB

class TextClassifier:
    def __init__(self):
        self.vectorizer = CountVectorizer()
        self.model = MultinomialNB()
        self._train_model()

    def _train_model(self):
        # Малък примерен dataset
        texts = [
            "I love this product",
            "This is amazing",
            "Very happy with the service",
            "I hate this",
            "This is terrible",
            "Very bad experience"
        ]

        labels = [
            "positive",
            "positive",
            "positive",
            "negative",
            "negative",
            "negative"
        ]

        X = self.vectorizer.fit_transform(texts)
        self.model.fit(X, labels)

    def predict(self, text):
        X = self.vectorizer.transform([text])
        return self.model.predict(X)[0]
